src/app/components/sandbox/CodeEditor/index.js

```diff
@@ -3,12 +3,19 @@ import Loadable from 'react-loadable';
 
 import Monaco from './Monaco';
 
+type Props = {
+  preferences: {
```

Comment:
You can use the type `Preferences` for this, it resides in `/common/types`.

Reply:
Nice, fixed.